CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+---
```

Comment:
I don't think we want this `---` at the top and at the bottom. The title should definitely be first.

Reply:
Resolved